Add tables for each Driver for security config via the URI

davidoliverSP2 opened the pull request:
This adds the tables for each Driver, for review purposes.

Note that the content of the tables will need to be amended when we merge forward

davidoliverSP2 committed:
add connection URI examples

davidoliverSP2 committed:
tidy up tables

gjmwoods reviewed (approved):
Looks good to me. Just wondering if we want a link from the connection-uri section back to these tables?

davidoliverSP2 committed:
reorg section

davidoliverSP2 committed:
add page level contents

davidoliverSP2 commented:
@gjmwoods @technige 

As discussed, I've reorganised the Client Applications section, added a page-level table of contents, and moved the connection settings table to a new "Examples" section.

Please let me know if this looks good to you.
